7997: Upgrade to Eclipse 2022-12

viragpurnam opened the pull request:
Upgrade to Eclipse 2022-12

<!-- Anything below this marker will be automatically updated, please do not edit manually! -->
---------
### Progress
- [x] Commit message must refer to an issue
- [ ] Change must be properly reviewed (1 review required, with at least 1 [Committer](https://openjdk.org/bylaws#committer))

### Issue
 * [JMC-7997](https://bugs.openjdk.org/browse/JMC-7997): Upgrade to Eclipse 2022-12


### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jmc pull/459/head:pull/459` \
`$ git checkout pull/459`

Update a local copy of the PR: \
`$ git checkout pull/459` \
`$ git pull https://git.openjdk.org/jmc pull/459/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 459`

View PR using the GUI difftool: \
`$ git pr show -t 459`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jmc/pull/459.diff">https://git.openjdk.org/jmc/pull/459.diff</a>

</details>

bric3 commented on releng/platform-definitions/platform-definition-2022-12/platform-definition-2022-12.target:
```diff
@@ -0,0 +1,69 @@
+            <repository location="http://download.eclipse.org/releases/2022-12/"/>
+        </location>
+        <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+            <unit id="org.eclipse.babel.nls_eclipse_ja.feature.group" version="4.22.0.v20211218020001"/>
+            <unit id="org.eclipse.babel.nls_eclipse_zh.feature.group" version="4.22.0.v20211218020001"/>
```
Is there a 4.26 version ?

thegreystone replied:
Nope, there isn't.

bric3 replied:
oh ok

thegreystone commented:
Haha. I did 7996 at the same time. :)

thegreystone commented:
Wanna take mine or yours @viragpurnam? I also deleted a few old (more than a year old) platforms, and fixed the project files in #460.

viragpurnam commented:
> Haha. I did 7996 at the same time. :)

:) Will take Yours 7996. I will close my PR.

viragpurnam commented:
> Wanna take mine or yours @viragpurnam? I also deleted a few old (more than a year old) platforms, and fixed the project files in #460.

Will take yours.. :) Later I will upgrade spifly version to 1.3.6

thegreystone commented:
Ok. Sounds good!

viragpurnam commented:
Duplicate of https://github.com/openjdk/jmc/pull/460
